pkg/server/handler.go

```diff
@@ -76,20 +70,20 @@ func (mc *MsgHandler) subscribeToMessages(msgChan chan network.InMsg) error {
 		err = mc.amqp.OnMessage(msgChan, queue, exchange, kind, key)
 	}
 
-	// Subscribe to messages received from any client
-	subscribe(msgChan, queueFogIn, exchangeFogIn, exchangeTypeFogIn, bindingKeyDevice)
-	subscribe(msgChan, queueFogIn, exchangeFogIn, exchangeTypeFogIn, bindingKeySchema)
-	subscribe(msgChan, queueFogIn, exchangeFogIn, exchangeTypeFogIn, bindingKeyDeviceCommands)
-	subscribe(msgChan, queueFogIn, exchangeFogIn, exchangeTypeFogIn, bindingKeyPublishData)
-
-	// Subscribe to messages received from the connector service
-	subscribe(msgChan, queueConnOut, exchangeConnOut, exchangeTypeConnOut, bindingKeyDataCommands)
-	subscribe(msgChan, queueConnOut, exchangeConnOut, exchangeTypeConnOut, bindingKeyDevice)
+	// Subscribe to general direct commands
+	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeyRegisterDevice)
+	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeyUnregisterDevice)
+	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeyRequestData)
+	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeyUpdateData)
+	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeySchemaSent)
 
 	// Subscribe to request-reply messages received from any client
 	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeyAuthDevice)
 	subscribe(msgChan, queueNameCommands, exchangeDevices, exchangeDevicesType, bindingKeyListDevices)
 
+	// Subscribe to broadcasted data events
+	subscribe(msgChan, queueNameEvents, exchangeDataSent, exchangeDataSentType, "")
```

Comment:
```suggestion
	subscribe(msgChan, queueNameEvents, exchangeDataSent, exchangeDataSentType, bindKeyAny)
```
Add other const value

Reply:
Done. I found `bindingKeyEmpty` better.